src/test/reload.test.ts

```diff
@@ -0,0 +1,137 @@
+import { Species } from "#app/enums/species.js";
+import { GameModes, getGameMode } from "#app/game-mode.js";
+import GameManager from "#test/utils/gameManager";
+import { afterEach, beforeAll, beforeEach, describe, expect, it } from "vitest";
+import { SPLASH_ONLY } from "./utils/testUtils";
+import { Moves } from "#app/enums/moves.js";
+import { EnemyCommandPhase } from "#app/phases/enemy-command-phase.js";
+import { DamagePhase } from "#app/phases/damage-phase.js";
```

Comment:
```suggestion
import { Moves } from "#app/enums/moves";
import { Species } from "#app/enums/species";
import { GameModes, getGameMode } from "#app/game-mode";
import GameManager from "#test/utils/gameManager";
import { afterEach, beforeAll, beforeEach, describe, expect, it } from "vitest";
import { SPLASH_ONLY } from "./utils/testUtils";
```
Removed the `.js` and now-unused imports (also my IDE re-ordered them but w/e).